components/organisms/CollaborationRequestWrapper/collaboration-requests-wrapper.tsx

```diff
@@ -34,6 +91,37 @@ const CollaborationRequestsWrapper = () => {
 
   return (
     <div>
+      <div className={clsx(
+        "flex flex-col sm:flex-row rounded-md border py-2 px-3 my-4 sm:space-x-2 space-y-2 sm:space-y-0 items-center",
+        emailPreference.receive_collaboration ? "border-green-500 bg-green-200" : "border-light-red-6 bg-light-red-4"
```

Comment:
I don't think the green border and color is needed. Keeping as grey is enough. But I'd argue that once enabled, we don't need the user turn off the feature from the profile. 

Once enabled, we can remove. 

The change i'd suggest is only show the red box when not enabled and not show the pending section until the feature is enabled.

Reply:
Very well! I get the point. 

I take it that the 'Red Box' displayed on the tab when collaboration is turned off would serve as notice for sure and sub-consciously encourages the contributors to enable collaboration.

Then the 'Green Box' removed, because we are not in-turn tryna get contributors to disable collaboration.

So I will approach like this...

- Re-appropriate all the utility classes to do away `green` colors
- Conditionally display the `div` using the initial api collab status response `insightsUser.receive_collaboration` 
```javascript
{!insightsUser?.receive_collaboration && (
  <div>
    ...
  </div>
)}
```

Reply:
But, I have this question @bdougie 

> ...and not show the pending section until the feature is enabled.

In cases where there are pending requests, are you suggesting that we should hide the `Collaboration Requests when `receive_collaboration` is `false`?